app/frontend/walletApp.js

```diff
@@ -76,6 +76,6 @@ Sentry.init({
 
 const Wrapper = h(Provider, {store}, h(App))
 
-loadWasm().then(() => {
+loadWasm().finally(() => {
```

Comment:
replaced .then with .finally since in case of rejected promise, whole page would not render. It makes more sense to me to render it anyway and catch the possible error somewhere else, with Sentry lets say.